src/kremling.d.ts

```diff
@@ -8,6 +8,7 @@ declare namespace Kremling {
   function m(className: string, condition: any): KremlingString & string;
   function toggle(truthyClass: string, falsyClass: string, condition: any): KremlingString & string;
   function t(truthyClass: string, falsyClass: string, condition: any): KremlingString & string;
+  function krem(strings: Array<string>, ...args: Array<string>): object;
```

Comment:
not 100% sure I did this right... it's been a while since I've written definitions

Reply:
this is valid. You can also do `string[]`